src/table/use-sticky-scrollbar.ts

```diff
@@ -124,10 +124,9 @@ export function useStickyScrollbar(
   }, [wrapperEl]);
 
   useEffect(() => {
-    if (supportsStickyPosition() && tableRef.current) {
-      const observer = new ResizeObserver(entries => {
+    if (supportsStickyPosition() && wrapperRef.current && tableRef.current) {
+      const observer = new ResizeObserver(() => {
         if (scrollbarContentRef.current) {
-          scrollbarContentRef.current.style.width = `${entries[0].borderBoxSize[0].inlineSize}px`;
```

Comment:
This line is not needed. We do the same here: https://github.com/cloudscape-design/components/pull/1056/files#diff-c2032c96c95559284e2956e45c92df9ff7e7f5960fab48f5cf3381ee7c528a56R70